api_db.php

```diff
@@ -71,7 +71,16 @@ function resolveHostname($clientip, $printIP)
 	$results = $db->query('SELECT * FROM network');
 
 	while($results !== false && $res = $results->fetchArray(SQLITE3_ASSOC))
+	{
+		$id = $res["id"];
+		$ipaddr = array();
+		$ips = $db->query("SELECT ip FROM network_addresses WHERE network_id = $id ORDER BY lastSeen DESC");
+		while($ips !== false && $ip = $ips->fetchArray(SQLITE3_ASSOC))
+			array_push($ipaddr,$ip["ip"]);
+		if(count($ipaddr) > 0)
+			$res["ip"] = $ipaddr;
```

Comment:
The response object should always have the IP array set, even if it is empty. It might even be better to cut out the middle variable (`$ipaddr`) and use `$res["ip"]` directly.